packages/mask/src/plugins/Collectible/SNSAdaptor/OpenSea/ListingByPriceCard.tsx

```diff
@@ -146,78 +151,77 @@ export function ListingByPriceCard(props: ListingByPriceCardProps) {
                     disableNativeToken={!paymentTokens.some((x) => isNativeTokenAddress(x.address))}
                     onAmountChange={setAmount}
                     onTokenChange={(x) => setAddress(x.address)}
-                    TokenAmountPanelProps={{
+                    ERC20InputProps={{
                         label: endingPriceChecked
                             ? t('plugin_collectible_starting_price')
                             : t('plugin_collectible_price'),
-                        TextFieldProps: {
-                            classes: {
-                                root: classes.panel,
-                            },
-                            helperText: endingPriceChecked
-                                ? t('plugin_collectible_set_initial_price')
-                                : t('plugin_collectible_ending_price_tip'),
-                        },
                     }}
                     FungibleTokenListProps={{
                         selectedTokens: token.value ? [token.value.address] : [],
                         tokens: paymentTokens,
                         whitelist: paymentTokens.map((x) => x.address),
                     }}
                 />
+                <Typography className={classes.helperText}>
```

Comment:
Can we make the helper text one of the properties of `<SelectTokenAmountPanel />`?

Reply:
We have used `<InputBase />` as the input component. It is already an atom level component. But, we can provide the `helperText` component.